Update main secrets documentation (#342)
spiceai/docs: spiceaidocs/docs/components/secret-stores/kubernetes/index.md

description: 'Kubernetes Secret Store Documentation'
---

> The `kubernetes` store allows Spice to read Kubernetes secrets by specifying the secret’s name with a selector.

Reviewer: ```suggestion
The `kubernetes` store supports reading specific secrets using a selector with the secret's name.
```
